Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM ruby:2.7.1-alpine3.11
+
+RUN apk update \
+    && apk upgrade \
+    && apk add build-base \
+               libffi-dev \
+               libxml2-dev \
+               libxslt-dev \
+               mariadb-dev \
+               mysql \
+               mysql-client \
+               ruby-dev \
+               build-base \
+               tzdata \
+               git
+
+RUN mkdir /myapp
```

Comment:
It would be a little more DRY (Don't Repeat Yourself) to set `/myapp` to a variable using `ARG` keyword, then reuse it.